src/nodes/Geometry3D/IndexedFaceSet.js

```diff
@@ -791,8 +791,21 @@ x3dom.registerNodeType(
                         hasColor = false;
                     }
                     this._mesh._numColComponents = numColComponents;
+                    if (fieldName.startsWith("attrib")) {
+                        var attrName = fieldName.slice(7);
```

Comment:
The idea of the new function updateAttrib(name) is not bad, it can be similar to the handleAttibs in X3domComposedGeometryNode.
The only problem is in IFS, I cannot imagine a (proper) way to know which attribute needs update. 
The ugly solution is to save in the node the name of the modified attribute (in _vf for exemple).